src/main/java/core/basesyntax/impl/PurchaseHandlerImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class PurchaseHandlerImpl implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
Avoid creating an instance of `FruitDaoImpl` inside the handler. Instead, pass it as a constructor parameter. This will also make your handler more independent and easier to test.